doc/PyFingerprint.rst

```diff
@@ -0,0 +1,15 @@
+PyFingerprint
```

Comment:
Why not Markdown? It is easier to read and to write

Reply:
I hate RST. But RST is the default for Sphinx. It is also possible to use Markdown for documentation but this is slightly more complicated (other Debian package dependency etc.)... Since we only use autodoc capabilities of Sphinx to create single HTML API documentation page it is not worth effort.